api-inference-community/docker_images/common/tests/test_api_audio_classification.py

```diff
@@ -39,7 +39,7 @@ def read(self, filename: str) -> bytes:
             bpayload = f.read()
         return bpayload
 
-    def test_simple(self):
+    def test_original_audiofile(self):
```

Comment:
Out of curiosity, what is the difference between naming conventions of: `test_simple` vs `test_original_audiofile`?

Reply:
`test_original_audiofile` is what's used in ASR, so I went with that one for consistency. (I now realize `audio-to-audio` is inconsistent with ASR. `audio-source-separation` is deprecated but due to the diff GitHub is marking as renaming

Reply:
I renamed these for consistency